src/nuclei/client/main.py

```diff
@@ -207,28 +269,36 @@ def get_endpoints(self, app: str) -> List[str]:
         ConnectionError:
             Application not available
         TypeError:
-            Wrong type for `app` argument
+            Wrong type for `app` or `version` argument
         ValueError:
-            Wrong value for `app` argument
+            Wrong value for `app` or `version` argument
         """
         if not isinstance(app, str):
             raise TypeError(
                 f"Expected positional argument `app` to be of type <class 'str'>, but got type: {type(app)}"
             )
 
-        return list(self._get_app_specification(app)["paths"].keys())
+        return list(self._get_app_specification(app, version)["paths"].keys())
 
-    def get_endpoint_type(self, app: str, endpoint: str) -> str:
+    def get_endpoint_type(
+        self, app: str, endpoint: str, version: str = "latest"
+    ) -> List[str]:
         """
+        Get HTTP methode used in endpoint.
```

Comment:
Description of the method seems to be outdated. It returns now a list of HTTP methodes?